diagnostic.md

```diff
@@ -36,6 +45,8 @@ Record your responses inside the fenced code blocks below each question.
 
     ```md
     <!-- your response here -->
+    the file structure of these two routes would be different. The first is a nested file system, the other flat
+    The first route is nested, the second route is not nested.
```

Comment:
You are correct that nested route definitions, do require a nested file structure.
Can you expand on this?
One of the biggest distinctions is where Ember will render the content.  The nested `'products'/'product'` instructs Ember to render `app/products/product/template.hbs` in the `outlet` of the parent `app/products/template.hbs`.  The non-nested `'product'` instructs Ember to render `app/product/template.hbs` in the context of the application template.  
These two route definitions are identical URL paths.